highfidelity/hifi
# Case 21746: Fix color grading/lighting of renderLayer front entities

https://highfidelity.fogbugz.com/f/cases/21746/renderLayer-front-entities-rendering-dark-black

Test plan:
- Run [this](https://gist.githubusercontent.com/SamGondelman/1b7418c416df22ddc84683b72e23a2fd/raw/fc5f092e8ca728e2c7a78c73c751d6f936c13b58/RenderLayer.js).
- You'll see this:
![image](https://user-images.githubusercontent.com/7650116/54458785-15b2a480-4722-11e9-85ec-09b88801eb25.png)
- Press "o".  The colors/shading on most of the entity types will not change (the polyvox entity might change slightly).  The objects will draw in front now:
![image](https://user-images.githubusercontent.com/7650116/54458862-3e3a9e80-4722-11e9-895e-770bde371e1e.png)
- Press "u".  The box, text, image, web, grid, and ring gizmo entities will become slightly transparent, but will still be lit the same way, and will still draw in front:
![image](https://user-images.githubusercontent.com/7650116/54458966-878aee00-4722-11e9-95d4-9a09328827ea.png)
